_src/static/images/edx-svg/style.css

```diff
@@ -0,0 +1,14 @@
+.icon {
```

Comment:
Sorry, which clean up PR? I checked https://github.com/edx/ux-pattern-library/pull/58, but that's JS-focused.